chainerx_cc/chainerx/cuda/memory_pool.cc

```diff
@@ -258,15 +258,15 @@ void MemoryPool::Free(void* ptr) {
     {
         std::lock_guard<std::mutex> lock{free_bins_mutex_};
 
-        // If the next chunk is free, merges it with them.
+        // If the next chunk is free, merges it with it.
```

Comment:
I don't understand `it with it` 😅

Reply:
`merges them`?

Reply:
`merge them` or `merges the chunk with the next chunk`?